### PR TITLE
User story 19

### PR DESCRIPTION
**Pull request by ggerdsen**
[x] done

User Story 19, Application Show Page

As a visitor
When I visit an applications show page "/applications/:id"
I can see the following:
- name
- address
- city
- state
- zip
- phone number
- Description of why the applicant says they'd be a good home for this pet(s)
- names of all pet's that this application is for (all names of pets should be links to their show page)

### CODE REVIEW

File: app/views/adoption_applications/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<hr>
+<h1><center>Application # <%= "#{@application.id}" %> </center></h1>
```
Comment (dhalverson) [inline app/views/adoption_applications/show.html.erb:2]: We can test for any nil edge cases to ensure the website works for all situations
